src/domain/event/EventRedirect.tsx

```diff
@@ -0,0 +1,164 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+import { useMutation, useQuery } from '@apollo/client';
+import * as Sentry from '@sentry/browser';
+
+import PageWrapper from '../app/layout/PageWrapper';
+import Text from '../../common/components/text/Text';
+import LinkButton from '../../common/components/button/LinkButton';
+import AnchorButton from '../../common/components/button/AnchorButton';
+import { ticketmasterEventQuery as TicketmasterEventQueryType } from '../api/generatedTypes/ticketmasterEventQuery';
+import { eventTicketmasterPasswordQuery } from './queries/eventQuery';
+import assignTicketSystemPasswordMutation from './mutations/assignTicketSystemPasswordMutation';
+import {
+  assignTicketSystemPasswordMutation as assignTicketSystemMutationData,
+  assignTicketSystemPasswordMutationVariables,
+} from '../api/generatedTypes/assignTicketSystemPasswordMutation';
+import styles from './eventRedirect.module.scss';
+import LoadingSpinner from '../../common/components/spinner/LoadingSpinner';
+import InfoPageLayout from '../app/layout/InfoPageLayout';
+import useGetPathname from '../../common/route/utils/useGetPathname';
+import TicketmasterPassword from './TicketmasterPassword';
+import Button from '../../common/components/button/Button';
+
+type Params = {
+  eventId: string;
+  childId: string;
+};
+
+const EventRedirect = () => {
+  const { t } = useTranslation();
+  const params = useParams<Params>();
```

Comment:
it would be better to destruct here


```suggestion
  const {eventId, childId} = useParams<Params>();
```